pkg/bmc/virtualmedia_validation.go

```diff
@@ -0,0 +1,117 @@
+package bmc
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/stmcginnis/gofish"
+	"github.com/stmcginnis/gofish/redfish"
+)
+
+// contains tells whether the array of redfish.VirtualMediaType
+// has the element redfish.VirtualMediaType.
+func contains(array []redfish.VirtualMediaType, element redfish.VirtualMediaType) bool {
```

Comment:
Go doesn't have a function for this? Oo I'm impressed.

Reply:
This function is going to be visible to everything else in this package, so it needs a more descriptive name.